x-pack/filebeat/input/azureeventhub/input_test.go

```diff
@@ -93,58 +99,6 @@ func TestProcessEvents(t *testing.T) {
 	assert.Equal(t, message, single)
 }
 
-func TestParseMultipleRecords(t *testing.T) {
```

Comment:
Why you removed those? Because you think that v1 is not going to be used anymore?